mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -478,6 +478,23 @@ func schemaNodeConfig() *schema.Schema {
 					ForceNew: true,
 					Description: `Setting this field will assign instances of this pool to run on the specified node group. This is useful for running workloads on sole tenant nodes.`,
 				},
+
+				"advanced_machine_features": &schema.Schema{
+					Type:        schema.TypeList,
+					Optional:    true,
+					MaxItems:    1,
+					Description: `Specifies options for controlling advanced machine features.`,
+					ForceNew:    false,
```

Comment:
This indicates that the field is updatable, which does't appear to be the case. I can't see a block in the message for node pool updates that makes it possible: https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters.nodePools/update

If this isn't updatable, we'll want to mark this field and `threads_per_core` as `ForceNew: true`. If it *is*, there'll be some extra work to make updating it work- GKE's complicated update methods require nonstandard code.

Reply:
Thanks, I see that there is no node config in the update API, so making it updateable is definitely out of scope of this change. Marking it with RequiredNew: true.